pycbc/mchirp_area.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+#
+# Integration of the area laying in the different cbc regions
+# By A. Curiel Barroso
+# August 2019
+
+"""This script computes the area corresponding to different CBC on the m1 & m2
```

Comment:
Since this isn't a script any more, better to say something like 'Functions to compute the area ..' 
Also, we don't need the `#!/usr/bin/env python` line at the beginning of the file as it's not an executable